packages/dropdowns/src/styled/items/StyledItem.ts

```diff
@@ -5,46 +5,79 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import styled from 'styled-components';
-import classNames from 'classnames';
-import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
-import MenuStyles from '@zendeskgarden/css-menus';
+import styled, { css, ThemeProps, DefaultTheme } from 'styled-components';
+import { retrieveComponentStyles, DEFAULT_THEME, getColor } from '@zendeskgarden/react-theming';
 
 const COMPONENT_ID = 'dropdowns.item';
 
 export interface IStyledItemProps {
-  isActive?: boolean;
   isFocused?: boolean;
-  isHovered?: boolean;
+  isCompact?: boolean;
   disabled?: boolean;
   checked?: boolean;
 }
 
+export const getItemPaddingVertical = (props: IStyledItemProps & ThemeProps<DefaultTheme>) => {
+  if (props.isCompact) {
+    return `${props.theme.space.base * 1.5}px`;
+  }
+
+  return `${props.theme.space.base * 2.5}px`;
+};
+
+export const getItemPaddingHorizontal = (props: IStyledItemProps & ThemeProps<DefaultTheme>) => {
+  if (props.isCompact) {
+    return `${props.theme.space.base * 6}px`;
+  }
+
+  return `${props.theme.space.base * 8}px`;
+};
+
+const getColorStyles = (props: IStyledItemProps & ThemeProps<DefaultTheme>) => {
+  return css`
+    background-color: ${props.isFocused &&
+      !props.disabled &&
+      getColor('primaryHue', 600, props.theme, 0.08)};
+    color: ${props.disabled
+      ? getColor('neutralHue', 400, props.theme)
+      : props.theme.colors.foreground};
+  `;
+};
+
 /**
- * Accepts all `<li>` props
- */
+ * 1. Allows an item to contain a positioned sub-menu.
+ * 2. Reset stacking context for sub-menu css-arrows.
+ **/
 export const StyledItem = styled.li.attrs<IStyledItemProps>(props => ({
   'data-garden-id': COMPONENT_ID,
   'data-garden-version': PACKAGE_VERSION,
-  className: classNames(MenuStyles['c-menu__item'], {
-    // State
-    [MenuStyles['is-active']]: props.isActive,
-    [MenuStyles['is-focused']]: props.isFocused,
-    [MenuStyles['is-hovered']]: props.isHovered,
-    [MenuStyles['is-disabled']]: props.disabled,
-    [MenuStyles['is-checked']]: props.checked
-  })
+  'aria-disabled': props.disabled
 }))<IStyledItemProps>`
-  /* stylelint-disable */
-  ${props =>
-    !props.isFocused &&
-    `&&&:hover {
-      background-color: inherit;
-    }
-  `};
+  display: block;
+  position: relative; /* [1] */
+  z-index: 0; /* [2] */
+  cursor: ${props => (props.disabled ? 'default' : 'pointer')};
+  padding: ${props => `${getItemPaddingVertical(props)} ${getItemPaddingHorizontal(props)}`};
+  text-decoration: none;
+  line-height: ${props => props.theme.space.base * 5}px;
```

Comment:
Could the [`getLineHeight`](https://github.com/zendeskgarden/react-components/blob/next/packages/theming/src/utils/getLineHeight.ts#L18) utility be used here?

Reply:
No, it needs pixel values in order to avoid cross-browser dimension shift.